coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -222,6 +226,35 @@ private void parseExcludeNodesFile(DataInputStream fsDataInputStream) throws IOE
         "Updated exclude nodes and {} nodes were marked as exclude nodes", excludeNodes.size());
   }
 
+  private boolean putInExcludeNodesFile(List<String> excludeNodes) throws IOException {
+    StringBuilder appendExecludeNodes = new StringBuilder();
+    String currentDate = DateFormatUtils.format(new Date(), DATE_PATTERN);
+    appendExecludeNodes
+        .append("# Header ")
+        .append(currentDate)
+        .append(":blacklist node added from the page.")
+        .append("\n");
+    for (String excludeNode : excludeNodes) {
+      appendExecludeNodes.append(excludeNode).append("\n");
+    }
+    appendExecludeNodes
+        .append("# End ")
+        .append(currentDate)
+        .append(":blacklist node added from the page.")
+        .append("\n");
+    Path hadoopPath = new Path(excludeNodesPath);
+    if (hadoopFileSystem != null) {
+      FileStatus fileStatus = hadoopFileSystem.getFileStatus(hadoopPath);
+      if (fileStatus != null && fileStatus.isFile()) {
+        try (FSDataOutputStream os = hadoopFileSystem.append(hadoopPath)) {
+          os.write(appendExecludeNodes.toString().getBytes(StandardCharsets.UTF_8));
```

Comment:
It's better to write to a temporary file first, and then rename it.

Reply:
> It's better to write to a temporary file first, and then rename it.

Now is the way to append content, temporary files must be read out first, this is really good?

Reply:
> > It's better to write to a temporary file first, and then rename it.
> 
> Now is the way to append content, temporary files must be read out first, this is really good?

I mean write to the file path like `hdfs://xxx/xxx/excludeFile.tmp` first, and then move `hdfs://xxx/xxx/excludeFile.tmp` to `hdfs://xxx/xxx/excludeFile`

Reply:
> > It's better to write to a temporary file first, and then rename it.
> 
> Now is the way to append content, temporary files must be read out first, this is really good?

Do you mean the feature just supports add servers to backlist and does not support remove them from backlist?

Reply:
I think it is better to replace all the content. We can display the content of the excludeFile on the front end and users can modify the contents at will. It is not easy to delete the excluded nodes from the file if append is used.

Reply:
> I think it is better to replace all the content. We can display the content of the excludeFile on the front end and users can modify the contents at will. It is not easy to delete the excluded nodes from the file if append is used.

done.